_pages/home.md

```diff
@@ -10,27 +10,43 @@ header:
   overlay_image: /assets/images/sorse-banner.svg
   overlay_filter: 0.5
   actions:
-    - label: Join the mailing list
-      url: https://www.listserv.dfn.de/sympa/subscribe/sorsenews
+    - label: Access event material
+      url: programme/
     - label: Watch past events
       url: https://www.youtube.com/channel/UCJ1CwxvODyT-eb4K7HfelGA
 excerpt: International Series of Online Research Software Events
-fullcalendar: true
 ---
 
 <aside id="twitter-holder" class="sidebar__right sticky">
     {% include twitter.html %}
 </aside>
 
+{% capture final-notice %}
+SORSE ended with its [final event]({{ "/programme/finale/" | relative_url }})
+on March 24th 2021. Thanks to all the contributors who submitted abstracts and
+gave presentations, discussions, demonstrations, and thanks to everyone who
+participated in one of our sessions to make all of this possible.
+
+But this won't be the end of everything. There is still need for exchange
+within our international community of research software engineers.
+
+Stay tuned and keep an eye on the communication channels of
+[your national RSE chapter]({{ "/contact/chapters/" | relative_url }}).
+{% endcapture %}
+
```

Comment:
Might it be worth adding an extra paragraph here - something for people who are in a country where there isn't yet a national community? For example:

"_If your country doesn't yet have a national RSE community to reach out to, get in touch and we can provide advice on international RSE groups that you can join or follow. If you're keen to start a national RSE community, we can put you in touch with members of established RSE communities who can offer advice._"

@sdruskat, I'd welcome your thoughts on this - I thought this could be a good way to help get people interested in starting new RSE associations in countries where they don't exist. I'm not sure where the "get in touch" link should point to at the moment - it could be the sorse.enquiries address but otherwise maybe the International Council?

Reply:
yes, I like the idea @jcohen02. We could link to https://researchsoftware.org/2018/04/18/how-to-setup.html

Reply:
or if that is already too involved, add a link to the [Council blog](https://researchsoftware.org/2021/01/27/introducing-the-international-council-of-RSE-associations.html) and mailing list (intl-rse-council@listserv.dfn.de)

Reply:
I'll merge this PR but please, @sdruskat, feel free to comment in this issue and we'll adapt the text later on